exe/iodine

```diff
@@ -80,7 +80,8 @@ module Iodine
           raise e
         end
         Iodine::Utils.monkey_patch ::Rack::Utils
-        ::Rack::Builder.parse_file(fn)
```

Comment:
We need to test for Rack versions here and choose the correct code path according to the correct `Rack::Builder` implementation... I can do it on my end.